packages/record-data/package.json

```diff
@@ -16,7 +16,8 @@
     "build": "ember build",
     "start": "ember serve",
     "test": "ember test",
-    "test:all": "ember try:each"
+    "test:production": "ember test -e production",
+    "test:try-one": "ember try:one --config-path='../-ember-data/config/ember-try.js'"
```

Comment:
use of `--config-path` here ensures all of the packages we add this command to share a single ember-try config for the various dep scenarios we would like to test.